docs/gettingstarted/index.md

```diff
@@ -82,3 +88,9 @@ Building and Testing Your Mod
 !!! note
 
     It is always advisable to test your mod in a dedicated server environment if it is intended to run there.
+
+
+!!! note
+
+    For `forge-1.12.2-14.23.5.2860-mdk` version, it looks like `gradlew runClient` will ignore all assets under `src/main/resourcs`. (BUG?)   
+
```

Comment:
Once again, no idea what this is about. I've verified it works fine, barring a few changes such that the build script looks like the 1.15 version.

Reply:
Temporarily deleted the note. I feel like this problem should discussed using github issue once.